inference-engine/src/vpu/common/include/vpu/utils/error.hpp

```diff
@@ -41,7 +41,7 @@ void throwFormat(const char* fileName, int lineNumber, const char* messageFormat
 #define VPU_THROW_UNLESS(condition, ...)                                                       \
     do {                                                                                       \
         if (!(condition)) {                                                                    \
-            vpu::details::throwFormat<vpu::details::VPUException>(__FILE__, __LINE__, __VA_ARGS__); \
+            ::vpu::details::throwFormat<::vpu::details::VPUException>(__FILE__, __LINE__, __VA_ARGS__); \
```

Comment:
Could you please explain why you need this?

Reply:
When I tried to add dynamicNMS transformation I've found namespace clashing between `vpu` and `ngraph::vpu` namespaces, so here I resolve it.